ElectronClient/app/gui/note-viewer/index.html

```diff
@@ -106,15 +106,13 @@
 
 		let previousContentHeight = contentElement.scrollHeight;
 		let startTime = Date.now();
-		ignoreNextScrollEvent = true;
```

Comment:
I did some testing and didn't find a use for them, it seems like the subsequent `restorePercentScroll();` even wasn't triggering a scroll event anyways, so by ignoring this event it introduced a subtle bug.